spring-integration-core/src/main/java/org/springframework/integration/config/GlobalChannelInterceptor.java

```diff
@@ -41,8 +41,8 @@
 public @interface GlobalChannelInterceptor {
 
 	/**
-	 * An array of simple patterns against which channel names will be matched. Default is "*"
-	 * (all channels). See {@link org.springframework.util.PatternMatchUtils#simpleMatch(String, String)}.
+	 * An array of patterns against which channel names will be matched. Default is "*"
+	 * (all channels). See {@link org.springframework.integration.util.PatternMatchUtils#smartMatch(String[], String)}.
```

Comment:
That `See` can move to the `@see` JavaDoc tag after `@return`.
Plus that would be great to mention in the general description a negation ability.